samples/csharp_dotnetcore/62.webex-adapter/Adapters/WebexAdapterWithErrorHandler.cs

```diff
@@ -11,7 +11,7 @@ namespace Microsoft.BotBuilderSamples.Adapters
     public class WebexAdapterWithErrorHandler : WebexAdapter
     {
         public WebexAdapterWithErrorHandler(IConfiguration configuration, ILogger<WebexAdapter> logger)
-            : base(configuration, logger)
+            : base(configuration, null, logger)
```

Comment:
This is the third time I've seen this change; was there a breaking change in the `Microsoft.Bot.Builder.Adapters.<channel-name>` adapters? Why didn't we create another `WebexAdapter()` overload that still took `IConfiguration` and `ILogger<T>` if the middle parameter is nullable?

Reply:
are you suggesting we add that extra constructor in 4.11.0 ❓ in which case we should get an issue open.
i made these changes so this compiles - given **4.10.0** is out the door I don't see what else we can do at this point. the change isn't breaking in this case because the channel adapters were actually preview (yes technically this is version one of the channel adapters 😄)